extension/src/popup/components/accountHistory/HistoryList/styles.scss

```diff
@@ -1,10 +1,10 @@
 .HistoryList {
-  max-height: 25rem;
-  padding-right: var(--popup--side-padding);
-  width: calc(var(--popup--width) - var(--popup--side-padding));
+  // max-height: 25rem;
```

Comment:
I'll do a cleanup for comments once we're sure the new layout functions well. I wanted to keep some code for reference in case we need to fix some things.